src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+    const last = this[this.length - 1];
+
+    for (const item of this) {
+      if (item === null || item === undefined) {
+        string += '';
```

Comment:
I think it is redundant

Reply:
I tried your option, but 4 tests are failed. Back to my solution.

Reply:
Because correct check will be:

```
      if (item !== null && item !== undefined) {
        string += item;
      }
```